runelite-client/src/main/java/net/runelite/client/plugins/banktags/BankTagsPlugin.java

```diff
@@ -157,6 +158,12 @@ public void shutDown()
 		shiftPressed = false;
 	}
 
+	private boolean isSearching()
+	{
+		return client.getVar(VarClientInt.INPUT_TYPE) == 11
```

Comment:
The next line is more self-explanatory, but what does `INPUT_TYPE == 11` represent? That should probably be a constant.

Reply:
True, turns out we even have an enum for it already.